x-pack/plugins/event_log/server/routes/find.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import {
+  IRouter,
+  RequestHandlerContext,
+  KibanaRequest,
+  IKibanaResponse,
+  KibanaResponseFactory,
+} from 'kibana/server';
+import { BASE_EVENT_LOG_API_PATH } from '../../common';
+import { findOptionsSchema, FindOptionsType } from '../event_log_client';
+
+const paramSchema = schema.object({
+  type: schema.string(),
+  id: schema.string(),
+});
+
+export const findRoute = (router: IRouter) => {
+  router.get(
+    {
+      path: `${BASE_EVENT_LOG_API_PATH}/{type}/{id}/_find`,
+      validate: {
+        params: paramSchema,
+        query: findOptionsSchema,
+      },
+    },
+    router.handleLegacyErrors(async function(
```

Comment:
I thought this just needed for Boom errors - which don't seem to be used here.  I guess we may need it to handle errors from the savedObjects client?

Reply:
Exactly... I added it after SO exploded :)